docs/rules/no-empty-fields.md

```diff
@@ -33,3 +33,22 @@ Example of **correct** code for this rule:
 	}
 }
 ```
+
+## Options
+
+### `ignoreProperties`
+
+Pass an array of top-level package properties to ignore.
+When provided, any errors related to the properties won't cause the rule to report a violation.
```

Comment:
I think @JoshuaKGoldberg commented on this, but I'm not seeing it anymore.  I generally agree with what he said, though.  Errors aren't errors if they're not reported 😁
Perhaps something along the lines of 

```suggestion
When provided, the rule won't report violations for the specified properties.
```